pex/resolver.py

```diff
@@ -47,46 +44,6 @@
     from pex.third_party import attr
 
 
-@attr.s
-class _ResolveHandler(object):
-    download_result = attr.ib()  # type: DownloadResult
-    wheel_builder = attr.ib()  # type: WheelBuilder
-    url_fetcher = attr.ib()  # type: URLFetcher
-    max_parallel_jobs = attr.ib(default=None)  # type: Optional[int]
-
-    def __call__(self, resolved_requirements):
-        # type: (Iterable[ResolvedRequirement]) -> None
-        self._resolved_requirements = resolved_requirements
-
-    def lock(self):
-        # type: () -> DownloadResult
-
-        build_requests = tuple(self.download_result.build_requests())
-        with TRACER.timed(
-            "Building {count} source {distributions} to gather metadata for lock.".format(
-                count=len(build_requests), distributions=pluralize(build_requests, "distribution")
-            )
-        ):
-            build_results = self.wheel_builder.build_wheels(
-                build_requests=build_requests,
-                max_parallel_jobs=self.max_parallel_jobs,
-            )
-            dist_metadatas = tuple(
-                DistMetadata.load(install_request.wheel_path)
-                for install_request in itertools.chain(
-                    tuple(self.download_result.install_requests()),
-                    itertools.chain.from_iterable(build_results.values()),
-                )
-            )
-        locked_resolve = LockedResolve.create(
```

Comment:
The removals in this file were incidental / should have happened in a prior refactor, Checking uses of `LockedResolve.create`, though, to see who passed platform_tag, led to discovering this dead use.